src/components/CategoryTabs/CategoryTabs.css

```diff
@@ -0,0 +1,110 @@
+.category-tabs-container {
+  position: relative;
+  max-width: 1000px;
+  max-height: 120px;
+  margin: 0 auto;
+  background-color: white;
+  z-index: 1000;
+}
+
+.category-tabs-wrapper {
+  overflow: hidden;
+  position: relative;
+}
+
+.category-tabs {
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+  padding: 10px 0;
+  border-bottom: 1px solid #ddd;
+  scroll-behavior: smooth;
+}
+
```

Comment:
To improve the .category-tabs style, I recommend removing the padding and replacing the border-bottom with a subtle shadow effect for a cleaner look:
.category-tabs {
  display: flex;
  overflow-x: auto;
  white-space: nowrap;
  /* padding: 10px 0; */
  /* border-bottom: 1px solid #ddd; */
  scroll-behavior: smooth;
}